### PR TITLE
Add HelpCommand

### PR DESCRIPTION
**Pull request by SomeHybrid**
Replaced DefaultHelpCommand with HelpCommand to help make -help look cooler

### CODE REVIEW

File: bot/util/help.py [hunk 1]
```diff
@@ -1,38 +1,10 @@
+from discord import commands
```
Comment (sexnine) [inline bot/util/help.py:1]: Should be importing from `discord.ext`

File: bot/util/help.py [hunk 2]
```diff
@@ -1,38 +1,10 @@
...
-            destination = self.get_destination(no_pm=True)
-            await destination.send("Couldn't send help to you due to blocked DMs...")
\ No newline at end of file
+class HelpCommand(commands.MinimalHelpCommand):
+    async def send_pages(self):
+        destination = self.get_destination()
+        embed = discord.Embed(color=discord.Color.yellow(), description='')
+        for page in self.paginator.pages:
+            embed.description += page
+        await destination.send(embed=e)
```
Comment (sexnine) [inline bot/util/help.py:10]: e is not defined.

**Review by sexnine — COMMENTED** [review]
@SomeHybrid Thank you for the PR :)

Just a tip... test your code before you submit a PR lol

File: bot/util/help.py (hunk 1 above)
Comment (SomeHybrid) [reply bot/util/help.py]: oh no i did a stupid dumb dumb

File: bot/util/help.py (hunk 2 above)
Comment (SomeHybrid) [reply bot/util/help.py]: oh no i did another stupid dumb dumb

**Pull request merged**